html/gui/js/libraries/PlotlyUtilities.js

```diff
@@ -161,10 +213,288 @@ function generateChartOptions(record, params) { // eslint-disable-line no-unused
+ * @return {Object} update - Layout object passed to Plotly.relayout
+ */
+function relayoutChart(chartDiv, adjHeight, firstRender = false, isExport = false) { // eslint-disable-line no-unused-vars
+    let update = {};
+    if (chartDiv._fullLayout.annotations.length > 0) {
+        const topCenter = topLegend(chartDiv._fullLayout);
+        const marginRight = chartDiv._fullLayout._size.r;
+        const marginLeft = chartDiv._fullLayout._size.l;
+        const legendHeight = (topCenter && adjHeight > 550) ? chartDiv._fullLayout.legend._height : 0;
+        let marginTop = legendHeight === 0 ? Math.max(chartDiv._fullLayout.margin.t, chartDiv._fullLayout._size.t)
+                                    : Math.min(chartDiv._fullLayout.margin.t, chartDiv._fullLayout._size.t);
+        let extraShift = 0;
+
+        const titleHeight = 31;
+        const subtitleHeight = 15;
+
+        let titleIndex = -1;
+        let subtitleIndex = -1;
+        let creditsIndex = -1;
+        let restrictedIndex = -1;
+        for (let idx = 0; idx < chartDiv._fullLayout.annotations.length; idx++) {
+            const { name } = chartDiv._fullLayout.annotations[idx];
+            switch (name) {
+                case 'title':
+                    titleIndex = idx;
+                    break;
+                case 'subtitle':
+                    subtitleIndex = idx;
+                    break;
+                case 'credits':
+                    creditsIndex = idx;
+                    break;
+                case 'Restricted Data Warning':
+                    restrictedIndex = idx;
+                    break;
+                default:
+            }
+        }
+
+        const isPie = chartDiv._fullData.length > 0 && chartDiv._fullData[0].type === 'pie';
+
+        const subtitleUpdates = adjustSubtitle(chartDiv._fullLayout, subtitleIndex, topCenter, firstRender);
+        update = subtitleUpdates.chartUpdates;
+
+        if (isPie && topCenter && subtitleUpdates.subtitleLineCount > 0) {
+            extraShift -= 10;
+        }
+
+        if (subtitleUpdates.subtitleLineCount > 0 && firstRender) {
+            marginTop = subtitleUpdates.chartUpdates['margin.t'];
+        }
+
+        if (topCenter && subtitleUpdates.subtitleLineCount === 0 && !isPie) {
+            extraShift += 15;
+        }
+
+        marginTop += extraShift;
+
+        const titleYShift = (marginTop + legendHeight) - titleHeight;
+
+        if (titleIndex !== -1) {
+            update[`annotations[${titleIndex}].yshift`] = subtitleUpdates.subtitleLineCount >= 3 ? titleYShift + 5 : titleYShift;
+        }
+
+        if (subtitleIndex !== -1) {
+            update[`annotations[${subtitleIndex}].yshift`] = titleYShift - (subtitleHeight * subtitleUpdates.subtitleLineCount);
+        }
+
+        const marginBottom = chartDiv._fullLayout._size.b;
+        let pieChartXShift = 0;
+        const exportShift = isExport ? 30 : 0;
```

Comment:
```suggestion
        const exportShift = isExport ? 30 : 0;
```
I would move this to right after the declaration of `const shiftYDown` to put it closer to where it is used.